src/snmpdevice/snmpdevicecfg.component.ts

```diff
@@ -118,7 +118,7 @@ export class SnmpDeviceCfgComponent {
   createStaticForm() {
     this.snmpdevForm = this.builder.group({
       ID: [this.snmpdevForm ? this.snmpdevForm.value.ID : '', Validators.required],
-      Host: [this.snmpdevForm ? this.snmpdevForm.value.Host : '', Validators.required],
+      Host: [this.snmpdevForm ? this.snmpdevForm.value.Host : '', Validators.compose([Validators.required, Validators.pattern("^([a-zA-Z0-9._-])+$")])],
```

Comment:
Instead of setting up here, would be interesting to set up as a new type into `ValidationService`  to be reusable for others components and return a message to the user (https://github.com/toni-moreno/snmpcollector/blob/master/src/common/validation.service.ts)